modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector.adoc

```diff
@@ -0,0 +1,115 @@
+
+== Prerequisites
+
+To use this connector, you should be familiar with Anypoint Studio, Anypoint connectors, and Mule flows. The Anypoint MQ connector is free, but connecting to the service in Anypoint Platform requires a license.
+
+=== Compatibility
+
+[%header%autowidth.spread]
+|===
+|Software |Version
+|Mule Runtime |4.1.1 and later
+|Anypoint Studio |v7 and later
+|===
+
+This documentation applies to Mule 4 only. For Mule 3 compatible connector information, see xref:3.9@mule-runtime::anypoint-mq-connector.adoc[Mule 3 MQ Connector].
+
+To migrate from the Mule 4 connector version 2.x to the Mule 4 connector version 3.x, see the xref:anypoint-mq/3.x/anypoint-mq-new-major.adoc[Anypoint MQ 3.x Connector Migration Guide].
+
+== Install Anypoint MQ Connector
+
+In Anypoint Studio:
+
+. Create or open a Mule project.
+. Click the Exchange *(X)* icon in the Studio task bar.
+. Enter your Anypoint Platform username and password and click *Sign in*.
+. Select *All assets* and search for "MQ"
+. Select "Anypoint MQ Connector - Mule 4", and click *Add to project*.
+. Follow the prompts to install the connector.
+
+For standalone apps or apps written in XML:
+
+. Open your Mule project in Anypoint Studio.
+. Add the connector as a dependency in the pom.xml file:
++
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>anypoint-mq-connector</artifactId>
+  <version>RELEASE</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+To specify a specific version, see the https://www.anypoint.mulesoft.com/exchange/com.mulesoft.connectors/anypoint-mq-connector/[Exchange Anypoint MQ Connector] asset. Click *Dependency Snippets* for a complete list.
+
+
+== Configure Anypoint MQ in a Mule Project
+
+In Anypoint Studio:
+
+. Install the connector, if it's not already installed. See <<Install Anypoint MQ Connector>>.
+. Create or open a Mule project.
+. Search for "mq" in the *Mule Palette* to see all the available operations for the *Anypoint MQ* connector.
+
+=== Create a New Configuration for the Anypoint MQ Connector
+
+. Drag the operation from the palette to the canvas, then select it.
+. Click the green plus sign to the right of *Connector configuration*.
+. In Anypoint Platform, click *MQ > Destinations*.
+. Copy the URL for your region to the *URL* field in Studio.
++
+. In Anypoint Platform, click *MQ > Client Apps*.
++
+If needed, create a client app.
+
+. Copy the *Client App ID* value to the *Client App ID* field in Studio.
+. Copy the *Client Secret* value to the *Client Secret* field in Studio.
++
+You can ignore the other settings to test your connector.
+. Click *Test Connection* to verify the Anypoint MQ Default subscriber.
+. Click *OK*.
+
+=== Select an Operation
+
+In the *Mule Palette* in Studio, click *Anypoint MQ* to display a list of available operations:
```

Comment:
This is Step 1 and should be numbered as such.
How are the operations listed in the UI? Can we show syntax or a screencap rather than this table?